src/Views/HardwareView.vala

```diff
@@ -191,6 +222,30 @@ public class About.HardwareView : Gtk.Grid {
         memory = GLib.format_size (get_mem_info ());
 
         // Graphics
+        try {
+            Process.spawn_command_line_sync ("lspci", out graphics);
```

Comment:
There should be another way of doing this as this is not portable across platforms (there is no lspci on ARM for instance)